varsfile.sample

```diff
@@ -5,3 +5,6 @@ ROUTE_SECRET_KEY_1: asdf
 ROUTE_SECRET_KEY_2: asdf
 AWS_ACCESS_KEY_ID: asdf
 AWS_SECRET_ACCESS_KEY: asdf
+BASIC_AUTH_USERNAME: asdf
+BASIC_AUTH_PASSWORD: asdf
+NOTIFY_LOG_LEVEL: asdf
```

Comment:
I think we should move this last line into `manifest.yml`, since we're trying to keep config there and only secrets in `varfile`